tests/integration/test_integration_metobs.py

```diff
@@ -113,11 +97,6 @@ def test_integration_metobs_first(
             raw_header_0
             header_0
         """
-        if raw_header_0:
-            raw_header_0 = raw_header_0.replace(
```

Comment:
If raw header isn't used any more, it could be deleted from the parametrisation above.

Reply:
Fixed in https://github.com/Ingenjorsarbete-For-Klimatet/ifk-smhi/pull/98/commits/bf5f6d2c6a51ae6ce0e3afdc9ddeb66846a73cd8